src/avram/base_query_template.cr

```diff
@@ -14,6 +14,12 @@ class Avram::BaseQueryTemplate
         end
       end
 
+      {% if named_args[:materialized_view] %}
+      def self.refresh_view(*, concurrent : Bool = false)
+        {{ type }}.database.exec("REFRESH MATERIALIZED VIEW #{concurrent ? "CONCURRENTLY" : ""} #{{{ type }}.table_name}")
+      end
```

Comment:
If you don't pass in this arg, should I still define the method, but raise an exception that tells you the model isn't defined as a materialzed view? :thinking: